src/client/driver/driver.js

```diff
@@ -210,7 +210,7 @@ export default class Driver {
     // We have to cancel every form submit after a test is done
     // to prevent requests to a closed session
     _onFormSubmit (e) {
-        if (this.contextStorage.getItem(TEST_DONE_SENT_FLAG))
+        if (this.contextStorage !== null && this.contextStorage.getItem(TEST_DONE_SENT_FLAG))
```

Comment:
I would prefer `if (this.contextStorage && this.contextStorage.getItem(...`, although your way is 100% correct in this case

Reply:
@AndreyBelym take a look on this PR please.
We need to check why the 'contextStorage' in not defined, however I think this PR can be merged now

Reply:
@AlexKamaev, @AndreyBelym See #4360, It's a race condition. In an IFrame, `driver.start()` is called on a promise, so a form that submits immediately triggers the `_onFormSubmit()` before `driver.start()` runs. For a main window this doesn't happen as `driver.start()` is called synchronously.